models/user_test.go

```diff
@@ -0,0 +1,45 @@
+package models
+
+import (
+	"testing"
+)
+
+func SetUser() *User {
+	return &User {
+		LoginName: "loginName",
+		GitHubUsername: "github_user",
+		SlackUsername: "slack_user",
+		SlackUserId: "SLACKID",
+	}
+}
+
+func TestSlackMention(t *testing.T) {
+	user := SetUser()
+
+	expect := "<@SLACKID|slack_user>"
+	actual := user.SlackMention()
+
+	if actual != expect {
+		t.Fatalf("%v does not much to expected: %v", actual, expect)
+	}
+}
+
+func TestEnvs(t *testing.T) {
+	user := SetUser()
+
+	expect := []string{
+		"GITHUB_USERNAME=github_user",
+		"SLACK_MENTION=<@SLACKID|slack_user>",
+	}
+	actual := user.Envs()
+
+	if len(actual) != len(expect) {
+		t.Fatalf("%v does not much to expected: %v", len(actual), len(expect))
+	}
+
+	for i := range expect {
+		if expect[i] != actual[i] {
+			t.Fatalf("%v does not much to expected: %v", actual, expect)
+		}
+	}
```

Comment:
To compare two slices, you can use [`reflect.DeepEqual`](https://golang.org/pkg/reflect/#DeepEqual). This checks whether two slices have the same elements or not.